app/components/Button/BlueButtonStyled.js

```diff
@@ -0,0 +1,12 @@
+import { blue, darkblue, transparent, white } from 'style-constants';
```

Comment:
It is better to not use color in components name for the following reasons:
1. There can be rebranding in the future and color may change. It should not be necessary to rename component in that case;
2. There may be a feature that allows user change theme preference

Name components based on role. E.g - primay button